backend/core/src/multiselect-question/multiselect-question.controller.ts

```diff
@@ -79,9 +82,21 @@ export class MultiselectQuestionsController {
     )
   }
 
+  @Get(`listings/:multiselectQuestionId`)
+  @ApiOperation({ summary: "Get multiselect question by id", operationId: "retrieve" })
+  async retrieveListings(
+    @Param("multiselectQuestionId") multiselectQuestionId: string
+  ): Promise<ListingDto[]> {
+    const results = await this.multiselectQuestionsService.findListingsWithMultiSelectQuestion(
+      multiselectQuestionId
+    )
+    return mapTo(ListingDto, results)
+  }
+
   @Delete()
   @ApiOperation({ summary: "Delete multiselect question by id", operationId: "delete" })
   @UseGuards(OptionalAuthGuard, AdminOrJurisdictionalAdminGuard)
+  @UseInterceptors(ActivityLogInterceptor)
```

Comment:
@emilyjablonski this is where the logging is happening. This interceptor will grab the applicable fields (user and preference id) and save it to the Activity Log table